cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -29,17 +29,30 @@
     <div class="eyebrow">{{ card.institution_name }}</div>
 
     <h1>{{ card.product_name }}</h1>
-
+    {% set credit_tiers = [
+        ("No credit score", "No score"),
+        ("619 or less", "619 or less"),
+        ("620 to 719", "620-719"),
+        ("720 or greater", "720+")
+    ] %}
```

Comment:
```suggestion
```

Was this accidentally added back in a merge after #8344?

Reply:
Ooh yeah, thanks. I had some nasty conflicts to resolve. I'll remove it again.